src/styles/index.scss

```diff
@@ -1,3 +1,231 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` style is applied to all elements by default in the code style guide, so you don't need to specify it for each class individually. You can set it globally for all elements using `* { box-sizing: border-box; }`. This will help to keep your CSS code DRY (Don't Repeat Yourself).